src/Entity/Pap/CampaignHistory.php

```diff
@@ -67,9 +68,11 @@
  *
  * @ApiFilter(SearchFilter::class, properties={
  *     "status": "exact",
+ *     "campaign.uuid": "exact"
  * })
  * @ApiFilter(AdherentIdentityFilter::class, properties={"questioner"})
  * @ApiFilter(DateFilter::class, properties={"createdAt", "beginAt"})
+ * @ApiFilter(OrderFilter::class, properties={"createdAt"})
```

Comment:
Why not already return a list sorted by creation date. That way there's no need to inject a filter.

Reply:
The order filter allow to change the order in the front, not only to set the order.

Reply:
ok